go.mod

```diff
@@ -40,9 +33,10 @@ replace (
 	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.31.1
```

Comment:
why does this still exist?

Reply:
I didn't want to remove any modules, do you think I should remove it?

Reply:
I think we need to find a command that does it for us

Reply:
I think my editor removing unused go modules, but not from the replace section

Reply:
strange

Reply:
I think the replace section is a section that you don't want that it will be edited automatically and you put it there in purpose.
Anyway do you want me to remove them?

Reply:
let's do it some other time

Reply:
ok I will open a ticket

Reply:
CSI-3360

Reply:
thanks